core/src/main/scala/org/bitcoins/core/protocol/script/ScriptPubKey.scala

```diff
@@ -1358,13 +1358,13 @@ object TaprootScriptPubKey extends ScriptFactory[TaprootScriptPubKey] {
                 s"Given asm was not a P2WSHWitnessSPKV0, got $asm")
   }
 
-  def apply(schnorrPubKey: SchnorrPublicKey): TaprootScriptPubKey = {
+  def apply(schnorrPubKey: XOnlyPubKey): TaprootScriptPubKey = {
```

Comment:
should rename variable

Reply:
Also would be nice to still have the ability to create from a `SchnorrPublicKey`